tests/installation/bootloader.pm

```diff
@@ -8,6 +8,12 @@ use Time::HiRes qw(sleep);
 sub run() {
     my ($self) = @_;
 
+    if (get_var("ARCH") eq 'aarch64') {
+        wait_serial('The highlighted entry will be executed automatically') || die "not found";
+        wait_serial("Booting `Installation'");
+        return;
+    }
```

Comment:
can you please put this in an extra bootloader_aarch64.pm instead of craming even more into this?